frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx

```diff
@@ -51,9 +51,7 @@ export function RobotStatusSection() {
     }, [enabledRobots, installationCode, switchSafeZoneStatus])
 
     const getRobotDisplay = () => {
```

Comment:
Does this need to be a function?
```suggestion
    const robotDisplay = robots.map((robot) => <RobotStatusCard key={robot.id} robot={robot} />)
```

Reply:
In general if a local function has no arguments (or if the argument values never change) and no side effects, then it can be safely made into a variable

Reply:
There are several like this so I suggest fixing them in separate PR